projects/js-packages/components/components/admin-page/index.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import JetpackAdminPageHeader from './header';
+import JetpackAdminPageFooter from './footer';
+
+/**
+ * This is the base structure for any admin page. It comes with Header and Footer.
+ *
+ * All content must be passed as children wrapped in as many <JetpackAdminSection> elements as needed.
+ *
+ * @param {object} props - Component properties.
+ * @returns {React.Component} JetpackAdminPage component.
+ */
+const JetpackAdminPage = props => {
```

Comment:
@leogermani, what do you think about moving `JetpackAdminPage`, `JetpackAdminPageHeader`, and `JetpackAdminPageFooter` components to the RNA Components package?
Since we're aiming at same base design for all consumers, should we reuse them for Jetpack plugins as well?